content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Multicluster Deployment Considerations
+description: Describes considerations when configuring your Multicluster Istio deployment.
+weight: 20
+keywords:
+  - multiple-clusters
+  - control-plane
+  - networks
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Before you install Istio on {{< gloss "multicluster" >}}multiple clusters{{< /gloss >}}, there are some things to
```

Comment:
Before you expand Istio to multiple clusters?

Reply:
That would be assuming that they're starting with a single cluster mesh. I think the current wording is slightly more generic in that it doesn't assume any particular starting state.